packages/grid/helm/syft/templates/backend/backend-statefulset.yaml

```diff
@@ -100,8 +104,6 @@ spec:
                 secretKeyRef:
                     name: {{ .Values.seaweedfs.secretKeyName | required "seaweedfs.secretKeyName is required" }}
                     key: s3RootPassword
-            - name: S3_PORT
```

Comment:
Q: Why did we remove the S3_PORT ?

Reply:
Probably, as there is no extra usefulness, is allowing users to customize this port?

Reply:
Not needed & not used. No point customizing the port as we're in a container environment and they can just port forward whatever to 8333

Reply:
You can still provide it as through `backend.env` in values.yaml